src/object/object_representation.c

```diff
@@ -1296,7 +1297,7 @@ or_varchar_length_internal (int charlen, int align)
     }
   else
     {
-      len = 1 + OR_INT_SIZE + charlen;
+      len = 1 + OR_INT_SIZE + OR_INT_SIZE + charlen;
```

Comment:
@lucianlazar2211 

What about adding a comment? 
It helps developers to easily understand its representation. :)